### ISSUE TITLE
A concept guide is incorrectly listed as a tutorial content type on the new Documentation index page

### ISSUE THREAD

**Issue by michelle-purcell** [quarkusio/quarkus#29727]
### Describe the bug

The following documentation page is listed on the Quarkus Documentation index page as a Tutorial.

https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/security-overview-concept.adoc

### Expected behavior

The page should display under the `Concepts` heading on the index page.

### Actual behavior

![image](https://user-images.githubusercontent.com/92924207/206164681-d02d37ca-1fd0-48e2-a05e-f14d2446e177.png)

### How to Reproduce?

Filename = [security-overview-concept.adoc](https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/security-overview-concept.adoc)
Header = 
```
[id="security-overview-concept"]
= Quarkus Security overview
include::_attributes.adoc[]
:categories: security, getting-started
```

### Additional information

This relates to the new landing page look and feel, currently in the `main` preview only. 

The problem is not relevant in the 2.14 and earlier production docs.

REF - [QDOCS-70](https://issues.redhat.com/browse/QDOCS-70) & [QDOCS-68](https://issues.redhat.com/browse/QDOCS-68)

**Comment by michelle-purcell** [comment]
@ebullient - Gotcha. Submitted a fix in PR #29742.  Thanks :pray:

**Comment by michelle-purcell** [comment]
@ebullient - I'll raise a separate PR to make this clear in the contributor guide.
